packages/windows/src/RPA/Windows/keywords/context.py

```diff
@@ -25,8 +25,9 @@ class ActionNotPossible(ValueError):
 class LibraryContext:
     """Shared context for all keyword libraries."""
 
-    def __init__(self, ctx):
+    def __init__(self, ctx, *args, **kwargs):
         self.ctx = ctx
+        self._locators_path = kwargs.get("locators_path", None)
```

Comment:
Since this is used inside of `LocatorKeywords` only, I would place it there only. (in `LocatorKeywords` class -- then no need for args/kwargs init idiom)

Reply:
That's a good point.